lib/charms/tls_certificates_interface/v1/tls_certificates.py

```diff
@@ -273,6 +274,7 @@ def _on_certificate_revoked(self, event: CertificateRevokedEvent) -> None:
 # to 0 if you are raising the major API version
 LIBPATCH = 11
```

Comment:
It's [already](https://charmhub.io/tls-certificates-interface/libraries/tls_certificates/source-code) 11 on ch so need to bump to 12.